src/uvw/stream.hpp

```diff
@@ -155,8 +155,8 @@ class StreamHandle: public Handle<T, U> {
 
 public:
 #ifdef _WIN32
```

Comment:
For the last commit, I need to request to revert all changes inside these blocks - I'll do that in a separate comment. Essentially nothing changes for MSVC. Otherwise, there's nothing wrong with it. The AppVeyor history for your repo has [one successful build for the same](https://ci.appveyor.com/project/skypjack/uvw/build/1.0.243).

If I understood @slyshykO's response correctly, it would break if we used the `#else` block (specifically line 162) for newer MSVC like in the second suggestion in my previous comment. Requesting @slyshykO to correct me if I got it wrong.

IMO, we can stick to the `#ifdef _MSC_VER` form if it is acceptable.
I checked it on my machine: the existing tests build and run with `#ifdef _MSC_VER`.